gym/envs/registration.py

```diff
@@ -589,15 +589,48 @@ def make(
         # Assume it's a string
         env_creator = load(spec_.entry_point)
 
-    # check render_mode is valid
-    render_modes = env_creator.metadata["render_modes"]
-    mode = kwargs.get("render_mode")
-    if mode is not None and mode not in render_modes:
-        raise error.Error(
-            f"Invalid render_mode provided: {mode}. Valid render_modes: None, {', '.join(render_modes)}"
-        )
+    mode = _kwargs.get("render_mode")
+    apply_human_rendering = False
+
+    # If we have access to metadata we check that "render_mode" is valid
+    if hasattr(env_creator, "metadata"):
+        render_modes = env_creator.metadata["render_modes"]
+
+        # We might be able to fall back to the HumanRendering wrapper if 'human' rendering is not supported natively
+        if (
+            mode == "human"
+            and "human" not in render_modes
+            and ("single_rgb_array" in render_modes or "rgb_array" in render_modes)
+        ):
+            logger.warn(
+                "You are trying to use 'human' rendering for an environment that doesn't natively support it. "
+                "The HumanRendering wrapper is being applied to your environment."
+            )
+            _kwargs["render_mode"] = (
+                "single_rgb_array"
+                if "single_rgb_array" in env_creator.metadata["render_modes"]
+                else "rgb_array"
+            )
+            apply_human_rendering = True
+        elif mode is not None and mode not in render_modes:
+            raise error.Error(
+                f"Invalid render_mode provided: {mode}. Valid render_modes: None, {', '.join(render_modes)}"
+            )
+
+    try:
+        env = env_creator(**_kwargs)
+    except TypeError as e:
+        if str(e).find("got an unexpected keyword argument 'render_mode'"):
+            raise AttributeError(
+                f"You passed render_mode='human' although {id} doesn't implement human-rendering natively. "
+                "Gym tried to apply the HumanRendering wrapper but it looks like your environment is using the old "
+                "rendering API, which is not supported by the HumanRendering wrapper."
```

Comment:
Would we want to include the actual error, just in case it is not this error?

Reply:
I think it should appear in the stacktrace

Reply:
Fair then ignore my comment